rules/badges/in_game.json

```diff
@@ -86,5 +86,13 @@
     "operation": "each_greater",
     "targets": [["kano-draw","groups.pixelhack.challengeNo", 13]],
     "title": "Pixel Hack"
+  },
+  "super_user": {
+    "bg_color": "FCE683",
+    "desc_locked": "Finish Terminal Quest to unlock this badge.",
+    "desc_unlocked": "You used your coding powers to rescue Folderton from the evil bell! You are truly a Super User.",
+    "operation": "each_greater",
+    "targets": [["linux-story", "finished", "challenge_46"]],
```

Comment:
Should this not be:

```
"targets": [["linux-story", "level", 46]]
```

like the above badge?

Reply:
For TQ, this `level` number means the latest completed challenge. And so, from 46 it would try to go to 47 when you click on "Continue Story". Given that 46 is the last challenge, that number doesn't progress past 45 such that it'll replay challenge 46 if you chose to "continue". So, I've saved & set a new variable when the last step in ch46 is completed to trigger this badge.